### PR TITLE
Create lezhin.com.xml

### PR DESCRIPTION
**Pull request by SaschaNaz**

### CODE REVIEW

**Comment by J0WI** [comment]
There are some more subdomains. Please also redirect the working and and comments for the broken ones.

WFM:
m.lezhin.com
link.lezhin.com

Mismatch:
social.lezhin.com
help.lezhin.com
ncdn.lezhin.com
cdn.lezhin.com

Refused:
desk.lezhin.com

File: src/chrome/content/rules/lezhin.com.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="lezhin.com">
+	<target host="lezhin.com" />
+	<target host="www.lezhin.com" />
+
+	<rule from="^http://lezhin.com" to="https://www.lezhin.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/lezhin.com.xml]: The `from` attribute is regex, you need to escape the dot.
Please also add a comment, that ^lezhin.com is refused over https.

**Comment by SaschaNaz** [comment]
The link.lezhin.com is not working for me and gives me HTTP 403 Unauthorized. Do you have any working URL for this?

File: src/chrome/content/rules/lezhin.com.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	help.lezhin.com
+	ncdn.lezhin.com
+	cdn.lezhin.com
+
+	Refused by HTTP 404:
+	desk.lezhin.com
+	link.lezhin.com
+	-->
+
+	<rule from="^http:\/\/lezhin\.com" to="https://www.lezhin.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/lezhin.com.xml]: You must not escape slashes.
Please also add trailing slashes to the url.

**Comment by J0WI** [comment]
Please move the comments on the top and give a hint for cause you redirect ^lezhin.com to www.

**Pull request merged**